src/views/Dashboard/Dashboard.js

```diff
@@ -10,15 +13,42 @@ import {
   LatestSales,
   LatestOrders
 } from './components';
+import { setUsersCount } from '../../redux/usersCountReducer';
+
 
 const useStyles = makeStyles(theme => ({
   root: {
     padding: theme.spacing(4)
   }
 }));
 
-const Dashboard = () => {
+const Dashboard = props => {
   const classes = useStyles();
+  const { getUsersAction, setUsersCountAction, usersCount } = props;
+
+  const getUsers = async () => {
+    const token = localStorage.getItem('jwtToken');
+
+    if (token) {
+      const response = await getUsersAction(token);
+
+      // Need to be admin in order to get 200 from /users
```

Comment:
This comment is not wrong? The api documentation says that everyone gets the status of 200 regardless of whether they have the role of admin or not.

Reply:
Ok, I understand the comment is ok, the documentation is not completed...